examples/minimal-appdir/src/server/uploadthing.ts

```diff
@@ -12,6 +12,7 @@ export const uploadRouter = {
     image: {
       maxFileSize: "4MB",
       maxFileCount: 4,
+      contentDisposition: "attachment",
```

Comment:
comment what this is about?

Reply:
I'll revert the example before merging - will add some words in docs instead